### DISCUSSION TITLE
Marimo notebook - Some ideas for future Positron Notebook

### DISCUSSION THREAD

**Discussion by ntluong95** [posit-dev/positron#5291] [Feedback]
Just came across it today and has not tested seriously yet, but I think an interactivity notebook would be the future!

https://docs.[marimo](https://docs.marimo.io/index.html).io/index.html

**Comment by sroener** [comment]
It would be great to add marimo as an optional drop-in replacement for Jupyter Notebooks. It gets somewhat closer to the feeling of executing R scripts step by step in R studio.
